### PR TITLE
full train fixes

### PR DESCRIPTION
**Pull request by cdoern**
- change LR to 2e-5 (learns better)
- honor the gradient accumulation given by the multipack sampler

the LR is the big change here. The previous version of full train has little to no improvement after 8 epochs. This version knows about the new information after as little as 1 epoch.

This is a major improvement. 

Also add logging to print information about the data

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the
  [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [ ] [Changelog](https://github.com/instructlab/instructlab/blob/main/CHANGELOG.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Functional tests have been added, if necessary.
- [ ] E2E Workflow tests have been added, if necessary.

### CODE REVIEW

**Review by jaideepr97 — APPROVED** [review]

**Review by Maxusmusti — APPROVED** [review]
Only improvements here, still more to do probably but these are all good changes

**Review by RobotSail — APPROVED** [review]
LGTM

**Comment by nathan-weinberg** [comment]
@Mergifyio backport release-v0.19
